middle_end/flambda2/simplify/simplify_binary_primitive.ml

```diff
@@ -86,7 +86,23 @@ end = struct
       | Prim of P.t
       | Exactly of N.Result.t
 
-    include Container_types.Make (struct
+    module Set : sig
+      type elt = t
+
+      type t
+
+      val empty : t
+
+      val add : elt -> t -> t
+
+      val is_empty : t -> bool
+
+      val cardinal : t -> int
+
+      val get_singleton : t -> elt option
+
+      val elements : t -> elt list
```

Comment:
Why is this signature explicit ? It doesn't seem to be needed (the `Set` module is not exported, and the signature is simple enough).